requirements.txt

```diff
@@ -0,0 +1,2 @@
+praw>=5.1.0
+pprint>=0.1
```

Comment:
👍  for adding a `requirements.txt` file.

Can we move this to a `etc/` folder?